src/components/RelicModal.tsx

```diff
@@ -208,6 +208,22 @@ export default function RelicModal(props: {
   }
   const handleOk = () => {
     relicForm.submit()
+    const currentValue = {
+      grade: relicForm.getFieldValue('grade'),
+      enhance: relicForm.getFieldValue('enhance'),
+      part: relicForm.getFieldValue('part'),
+      set: relicForm.getFieldValue('set'),
+      mainStatType: relicForm.getFieldValue('mainStatType'),
+      substatType0: relicForm.getFieldValue('substatType0'),
+      substatValue0: relicForm.getFieldValue('substatValue0'),
+      substatType1: relicForm.getFieldValue('substatType1'),
+      substatValue1: relicForm.getFieldValue('substatValue1'),
+      substatType2: relicForm.getFieldValue('substatType2'),
+      substatValue2: relicForm.getFieldValue('substatValue2'),
+      substatType3: relicForm.getFieldValue('substatType3'),
+      substatValue3: relicForm.getFieldValue('substatValue3'),
+    }
+    if (compareRelics(props.selectedRelic, currentValue)) props.selectedRelic.verified = false
```

Comment:
This feels like the wrong place to do this check, this should happen after the validation but before the relic is saved to the DB

For example, if we edit a valid verified relic, make an invalid change, this would show an error "Substats are invalid/etc", not make the change, but still unverify the relic.

`relicForm.submit()` should trigger the `onFinish` function and I think right before `console.log('Completed relic', relic)` is probably the right place